### DISCUSSION TITLE
Bluetooth audio stops or gets interrupted when turning head

### DISCUSSION THREAD

**Discussion by rekredhead** [blueman-project/blueman#2453] [Q&A]
blueman:
BlueZ:
Distribution: Linux Mint 21.3
Desktop environment: Cinnamon

* [x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

Description: I am able to connect my Bluetooth earbuds (JBL WaveBeam) to my laptop (ASUS TUF F15) in Linux Mint. By default, I'm connected to A2DP Sink and I can hear the audio clearly. But when I turn my head by around 45 degrees while the earbuds are connected, the audio stops. When I turn my head back towards my screen, the audio comes back but the quality is greatly reduced for a short while until it goes back to normal.
I then switched to the HFS audio profile and I had no issues when I'm turning my head. But the audio quality is very bad in that mode.

Clarifications: There no issues with my earbuds or my laptop. I connected the earbuds to my phone and faced no such issues. I connected the earbuds to my laptop on Windows 10 (I'm using dual-boot atm) and faced no such issues.

My Issues:
1. Is it possible to use the A2DP Sink audio profile while I'm still able to move around freely when the earbuds attached my ears?
2. Is it possible to increase the audio quality of HFS so I can use it instead of A2DP Sink?
3. Can I know if there is a fix for this or if it's an issue that needs to be addressed?

Also, my thanks to the team for the all the effort put into this project.

**Comment by cschramm** [comment]
Anything logged by e.g. the kernel, bluetoothd or your audio server (pulseaudio, I guess)?
